CONTRIBUTING.md

```diff
@@ -127,15 +127,31 @@ for each package and automatically generate the changelog by adhering to [Conven
 [lerna-mode]: https://github.com/lerna/lerna#independent-mode
 [cc]: https://www.conventionalcommits.org/en/v1.0.0/
 
+### Prerelease testing
+
+You can test the CI flow using [act](https://github.com/nektos/act). Follow the installation process and use command
+
+#### Pull Requests
+
+```
+act --detect-event  pull-request
+```
+
+#### Complete push to master
+
+Prerequisite: Create a folder /tmp/ghartifacts or change the path below
+
+```
+act --detect-event  push --artifact-server-path /tmp/ghartifacts
+```
+
 ### Release Process
 
 Performing a release requires push permissions to the repository.
 
 - Ensure you are on `master` branch and synced with origin.
 - `yarn run release:version`
 - Follow the lerna CLI instructions.
-- Track the new `/v\d+.\d+.\d+/` tag build on circle-ci.
```

Comment:
should not this step be replaced (instead of removed) with tracking the build of the relevant github action?